omod/src/main/webapp/resources/js/view/list.js

```diff
@@ -0,0 +1,163 @@
+                    }, self);
+
+                    item.editor.on('change', function () {
+                        if (!item.addEventTriggered) {
+                            item.addEventTriggered = true;
+                            this.trigger('add', this, item.editor);
+                        }
+                        this.trigger('item:change', this, item.editor);
+                        this.trigger('change', this);
+                    }, self);
+
+                    item.editor.on('focus', function () {
+                        if (this.hasFocus) {
+                            return;
+                        }
+
+                        this.trigger('focus', this);
+                    }, self);
+                    item.editor.on('blur', function () {
+                        if (!this.hasFocus) {
+                            return;
+                        }
+
+                        var self = this;
+                        setTimeout(function () {
+                            if (_.find(self.items, function (item) {
+                                    return item.editor.hasFocus;
+                                })) {
+                                return;
+                            }
+
+                            self.trigger('blur', self);
+                        }, 0);
+                    }, self);
+
+                    if (userInitiated || value) {
+                        item.addEventTriggered = true;
+                    }
+
+                    if (userInitiated) {
+                        self.trigger('add', self, item.editor);
+                        self.trigger('change', self);
+                    }
+                };
+
+                //Check if we need to wait for the item to complete before adding to the list
+                if (this.Editor.isAsync) {
+                    item.editor.on('readyToAdd', _addItem, this);
+                } else {
+                    //Most editors can be added automatically
+                    _addItem();
+                }
+
+                return item;
+            }
+        });
+
+        /**
+         * A single item in the list
+         *
+         * @param {editors.List} options.list The List editor instance this item belongs to
+         * @param {Function} options.Editor   Editor constructor function
+         * @param {String} options.key        Model key
+         * @param {Mixed} options.value       Value
+         * @param {Object} options.schema     Field schema
+         */
+        editors.InventoryList.Item = editors.List.Item.extend({
+            events: {
+                'click [data-action="remove"]': function (event) {
+                    event.preventDefault();
+                    var name = this.key;
+                    name =name.substring(0, name.length -1);
+                    var message='Are you sure you want to delete this';
+                    if(name == 'price'){

```

Comment:
Rather than having this if/else statement, couldn't you look at the model schema and/or meta information to figure out the title for the thing to be removed?